lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -51,10 +51,10 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     % if facebook_share_enabled:
                       <button class="action action-share-facebook btn-inverse btn-small icon-only" id="action-share-facebook"
                          onclick="FaceBook.share({
-                          share_text: '${facebook_share_text | n, js_escaped_string}', ## xss-lint: disable=mako-invalid-html-filter
-                          share_link: '${share_url | n, js_escaped_string}', ## xss-lint: disable=mako-invalid-html-filter
-                          picture_link: '${full_course_image_url | n, js_escaped_string}', ## xss-lint: disable=mako-invalid-html-filter
-                          description: '${_('Click the link to see my certificate.') | n, js_escaped_string}' ## xss-lint: disable=mako-invalid-html-filter
+                          share_text: '${facebook_share_text | n, js_escaped_string}', 
+                          share_link: '${share_url | n, js_escaped_string}',
+                          picture_link: '${full_course_image_url | n, js_escaped_string}',
+                          description: '${_('Click the link to see my certificate.') | n, js_escaped_string}'
```

Comment:
Won't this result in linting alerts?

Reply:
Here is the linting error: https://github.com/openedx/edx-platform/runs/6615845706?check_suite_focus=true#step:11:75

Can you see if there is some other way to make the linting exception?